javascript/apps/taiga/src/app/modules/project/story-detail/data-access/+state/reducers/story-detail.reducer.ts

```diff
@@ -276,6 +276,24 @@ export const reducer = createImmerReducer(
         state.totalComments = 1;
       }
 
+      return state;
+    }
+  ),
+  on(
+    StoryDetailApiActions.deleteCommentSuccess,
+    (state, { commentId, deletedBy, deletedAt }): StoryDetailState => {
+      state.comments = state.comments.map((comment) =>
+        comment.id === commentId
+          ? {
+              ...comment,
+              deletedBy,
+              deletedAt: deletedAt || new Date().toISOString(),
+            }
+          : comment
+      );
+
+      if (state.totalComments) state.totalComments = state.totalComments - 1;
```

Comment:
Does the lintern allow this style? As far as i know we always write it like as it follows:
```
if (state.totalComments) {
state.totalComments = state.totalComments - 1;
}
```